src/Dev/DevelopmentAdmin.php

```diff
@@ -190,6 +176,33 @@ protected static function get_links()
         return $links;
     }
 
+    protected function getLinks(): array
+    {
+        $canViewAll = $this->canViewAll();
+        $links = [];
+        $reg = Config::inst()->get(static::class, 'registered_controllers');
+        foreach ($reg as $registeredController) {
+            if (isset($registeredController['links'])) {
+                if (!ClassInfo::exists($registeredController['controller'])) {
+                    continue;
+                }
```

Comment:
Why is this necessary? It isn't in the `get_links` implementation.

Reply:
Otherwise we try to call `->hasMethod()` on `null`

Reply:
Let me step back a bit then - in what scenario are we getting a value in `$registeredController['controller']` which isn't a class?

Reply:
[In the tests](https://github.com/silverstripe/silverstripe-framework/blob/5/tests/php/Dev/DevAdminControllerTest.php#L32), actually 😅

Reply:
Oh! Love that lol. Yeah in that case it's good that we're now actually explicitly checking this - it looks like it should have been that way the whole time, given that test case.